src/routers/employer.ts

```diff
@@ -11,5 +13,7 @@ const router = express.Router()
 router.post('/login/local', localLogin)
 router.post('/create', registerEmployer)
 router.post('/jobs/:companyName', createJobPost)
+router.get('/', getEmployer)
```

Comment:
You have to provide an employer id as a param to get that specific employer

Reply:
That's the router for getting all the employers.

Reply:
I change the name to getEmployers